run_service.py

```diff
@@ -57,10 +57,10 @@ def unit_to_wei(unit: float) -> int:
     """Convert unit to Wei."""
     return int(unit * 1e18)
 
-WALLET_TOPUP = unit_to_wei(0.05)
-MASTER_SAFE_TOPUP = unit_to_wei(0.01)
-SAFE_TOPUP = unit_to_wei(0.02)
-AGENT_TOPUP = unit_to_wei(0.01)
+WALLET_TOPUP = unit_to_wei(0.005)
+MASTER_SAFE_TOPUP = unit_to_wei(0.001)
+SAFE_TOPUP = unit_to_wei(0.002)
+AGENT_TOPUP = unit_to_wei(0.001)

```

Comment:
Is this feasible @dvilelaf ? .. I am just trying to get the funding requirements into a more reasonable area

Reply:
I think so